src/include/migraphx/op/nonmaxsuppression.hpp

```diff
@@ -184,13 +185,20 @@ struct nonmaxsuppression
             result.y = {static_cast<double>(start[0]), static_cast<double>(start[2])};
         }
 
+        result.sort();
```

Comment:
Can half_width and half_height be negative.? This sort() should move into the else clause above. Thanks.

Reply:
sort() sorts the coordinates of the box result given back since a coordinates for each box is in ascending order.

Reply:
Agreed. Sorted result is what you automatically get - in the if clause. Thanks.

Reply:
You need to sort for both cases because you can't have a negative area after creating a batch box from the stored coordinates in the index of boxes. 

Having the sort at the end ensures ordering of start coordinates doesn't matter as they may be different based on how the graph was generated and converted

Reply:
Add doc spec from Onnx to clear this up.